pages/common/pio-settings.md

```diff
@@ -0,0 +1,20 @@
+# pio settings
+
+> View and modify PlatformIO settings.
+> More information: <https://docs.platformio.org/en/latest/core/userguide/cmd_settings.html>.
+
+- List all PlatformIO settings:
+
+`pio settings get`
+
+- Show a specific PlatformIO setting:
+
+`pio settings get {{setting}}`
+
+- Modify a setting:
```

Comment:
```suggestion
- Modify the value of a setting (creates the setting if it does not exist):
```

Reply:
It doesn't create a new setting, just prints an error

Reply:
👍🏻 check again